inc/saihostif.h

```diff
@@ -736,9 +739,11 @@ typedef enum _sai_hostif_attr_t
      * The maximum number of characters for the name is SAI_HOSTIF_NAME_SIZE - 1 since
      * it needs the terminating null byte ('\0') at the end.
      *
+     * If Hostif is a generic netlink, this indicates the generic netlink family name.
+     *
      * @type char
      * @flags MANDATORY_ON_CREATE | CREATE_ONLY
-     * @condition SAI_HOSTIF_ATTR_TYPE == SAI_HOSTIF_TYPE_NETDEV
+     * @condition SAI_HOSTIF_ATTR_TYPE == SAI_HOSTIF_TYPE_NETDEV or SAI_HOSTIF_ATTR_TYPE == SAI_HOSTIF_TYPE_GENETLINK
```

Comment:
Is it always equal to "NETLINK_GENERIC"?

Reply:
This is set to the generic netlink family name whose multicast port id (SAI_HOSTIF_ATTR_GENETLINK_PORT_ID) is used to lift the samples. If the psample driver is used, it should be set to "psample" (refer https://github.com/torvalds/linux/blob/master/include/uapi/linux/psample.h#L33 ).